ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2081,3 +2081,33 @@ def test_torch_sigmoid(
         on_device=on_device,
         input=x[0],
     )
+
+
+# lerp
+@handle_frontend_test(
+    fn_tree="torch.lerp",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=3,
```

Comment:
Add the following:
`       large_abs_safety_factor=2.5,
        small_abs_safety_factor=2.5,
        safety_factor_scale="log",`
Some large values will not interpolate properly due to float percision limitations, this prevents those from being generated